roles/rsyslog/tasks/inputs/files/main.yml

```diff
@@ -16,10 +16,10 @@
         type: input
         state: "{{ item.state | d('present') }}"
         sections:
-          - options: 'input(type="imfile"
-            file="{{ item.input_log_path }}" tag="{{ item.name }}")
-
-            {{ lookup("template", "input_template.j2") }}'
+          - options: >-
```

Comment:
I think this would be better to use `|-` - what you have works because `>-` knows how to handle the indentation of the `tag` line correctly, and the template expansion happens after `>-` processes the string (or `>-` preserves the newlines in the template).  Using `|-` makes it explicit that you want the line breaks and the indentation preserved as-is.